wagtail_newsletter/models.py

```diff
@@ -133,3 +140,23 @@ def save_revision(self, *args, **kwargs):
             clean=False,
         )
         return revision
+
+    newsletter_template: str
+
+    def get_newsletter_template(self) -> str:
+        return self.newsletter_template
+
+    def get_newsletter_context(self) -> "dict[str, Any]":
+        return {"page": self}
+
+    def get_newsletter_html(self):
```

Comment:
```suggestion
    def get_newsletter_html(self) -> str:
```

(yup, that's the epitome of nitpicking :sweat_smile: )

Reply:
Yes, I'm leaving out one type annotation per PR, so that you can flag it 😆🙈 Srsly though, eagle eyes, nice catch!

I've debated with myself whether to annotate methods from Wagtail, e.g. `serve_preview()`, but that seems counterproductive, since they're not annotated upstream, and won't typically be called by downstream code. Makes sense, do you think?